ArduCopter/RC_Channel.cpp

```diff
@@ -703,68 +709,78 @@ void RC_Channel_Copter::do_aux_function_change_force_flying(const AuxSwitchPos c
     }
 }
 
+// note that this is a method on the RC_Channels object, not the
+// individual channel
 // save_trim - adds roll and pitch trims from the radio to ahrs
-void Copter::save_trim()
+void RC_Channels_Copter::save_trim()
 {
     // save roll and pitch trim
-    float roll_trim = ToRad((float)channel_roll->get_control_in()/100.0f);
-    float pitch_trim = ToRad((float)channel_pitch->get_control_in()/100.0f);
-    ahrs.add_trim(roll_trim, pitch_trim);
+    float roll_trim = ToRad((float)get_roll_channel().get_control_in()/100.0f);
+    float pitch_trim = ToRad((float)get_pitch_channel().get_control_in()/100.0f);
+    AP::ahrs().add_trim(roll_trim, pitch_trim);
     LOGGER_WRITE_EVENT(LogEvent::SAVE_TRIM);
     gcs().send_text(MAV_SEVERITY_INFO, "Trim saved");
 }
 
+#if AP_COPTER_AHRS_AUTO_TRIM_ENABLED
+
+void RC_Channels_Copter::do_aux_function_ahrs_auto_trim(const RC_Channel::AuxSwitchPos ch_flag)
```

Comment:
let's put a comment above this function to explain what it's doing.  Somethjing like, "start/stop ahrs auto trim"